src/popup.css

```
@@ -0,0 +1,462 @@
+  --color-outline-rgb: 153, 162, 160;
+  --color-accent-border: #3a524b;
+  --color-link-icon: #7cc293;
+  --color-status: #d9b38c;
+}
+
+/* ===== 字体 =====
+   品牌字体为构建期生成的 UI 文案子集（scripts/subset-font.mjs），
+   仅覆盖固定 UI 文案字符，因此只应用于 UI 框架（标题/按钮/状态），
+   用户数据（书签树、搜索输入）使用系统字体栈渲染。 */
+@font-face {
+  font-family: "AlibabaHealthFont";
```

Stylelint (17.14.0)

[error] Expected no quotes around "AlibabaHealthFont" (font-family-name-quotes)

(font-family-name-quotes)